wallet/src/de/schildbach/wallet/ui/WalletTransactionsFragment.java

```diff
@@ -391,7 +391,7 @@ protected CharSequence collectWalletDump() {
     public void onTransactionRowClicked(Transaction tx) {
         TransactionDetailsDialogFragment transactionDetailsDialogFragment =
                 TransactionDetailsDialogFragment.newInstance(tx.getTxId());
-        transactionDetailsDialogFragment.show(getChildFragmentManager(), null);
+        transactionDetailsDialogFragment.show(getParentFragmentManager(), null);
```

Comment:
We should show dialogs with `parentFragmentManager` so that it is dismissable from the `LockScreenActivity`.